setup.py

```diff
@@ -6,21 +6,44 @@
 
 setup(
     name="Open edX",
-    version="0.2",
-    install_requires=['distribute'],
+    version="0.3",
```

Comment:
I suspect this might be up to the Open Source team, actually. I know we've only done two named releases as of right now, so it'll be good for us to check with them to make sure that upping the version number isn't going to be a problem for other people who are using our code.